src/java/org/apache/cassandra/cql3/functions/FunctionParameter.java

```diff
@@ -115,66 +115,11 @@ public String toString()
         };
     }
 
-    /**
-     * @return a function parameter definition that accepts values of type {@link CollectionType}, independently of the
-     * types of its elements.
-     */
-    public static FunctionParameter anyCollection()
-    {
-        return new FunctionParameter()
-        {
-            @Override
-            public void validateType(FunctionName name, AssignmentTestable arg, AbstractType<?> argType)
-            {
-                if (!argType.isCollection())
-                    throw new InvalidRequestException(format("Function %s requires a collection argument, " +
-                                                             "but found argument %s of type %s",
-                                                             name, arg, argType.asCQL3Type()));
-            }
-
-            @Override
-            public String toString()
-            {
-                return "collection";
-            }
-        };
-    }
-
-    /**
-     * @return a function parameter definition that accepts values of type {@link SetType} or {@link ListType}.
-     */
-    public static FunctionParameter setOrList()
-    {
-        return new FunctionParameter()
-        {
-            @Override
-            public void validateType(FunctionName name, AssignmentTestable arg, AbstractType<?> argType)
-            {
-                if (argType.isCollection())
-                {
-                    CollectionType.Kind kind = ((CollectionType<?>) argType).kind;
-                    if (kind == CollectionType.Kind.SET || kind == CollectionType.Kind.LIST)
-                        return;
-                }
-
-                throw new InvalidRequestException(format("Function %s requires a set or list argument, " +
-                                                         "but found argument %s of type %s",
-                                                         name, arg, argType.asCQL3Type()));
-            }
-
-            @Override
-            public String toString()
-            {
-                return "numeric_set_or_list";
-            }
-        };
-    }
-
     /**
      * @return a function parameter definition that accepts values of type {@link SetType} or {@link ListType},
      * provided that its elements are numeric.
```

Comment:
nit: need to update the java doc since it now accepts number type too.

Reply:
Right, updated.